documentation/book/proc-replacing-private-keys.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-replacing-private-keys-{context}']
+
+= Replacing private keys
+
+You can replace the private keys used by the cluster CA and clients CA certificates. 
+When a private key is replaced, the Cluster Operator generates a new CA certificate for the new private key.
+
+.Prerequisites
+
+* The Cluster Operator is running.
+* A Kafka cluster in which CA certificates and private keys are installed.
+* One or more maintenance time windows are configured for the Kafka cluster.
+
+.Procedure
+
+* Apply the `strimzi.io/force-replace` annotation to the `Secret` that contains the private key that you want to renew.
+
++
+[cols="3*",options="header",stripes="none",separator=¦]
+|===
+
+¦Private key for
+¦Secret
+¦Annotate command
+
+¦Cluster CA
+¦_<cluster-name>_-cluster-ca
+m¦kubectl annotate secret _<cluster-name>_-cluster-ca strimzi.io/force-replace=true
+
+¦Clients CA
+¦_<cluster-name>_-clients-ca
+m¦kubectl annotate secret _<cluster-name>_-clients-ca strimzi.io/force-replace=true
+
+|===
+
+If a maintenance time window is active, at the next reconciliation the Cluster Operator:
```

Comment:
As above - this is good. But I think I would add one more sentence at the end of this paragraph. Something like _If you are not using the maintenance time window, the new CA will be generated in the next reconciliation._ since right now it could be confusing for users who do not use the maintenance time window at all.

Reply:
Ditto as above.